composer.json

```diff
@@ -4,16 +4,19 @@
     "type": "library",
     "license": "proprietary",
     "require": {
-        "php": "^7.1",
+        "php": "^7.3|^8.0",
```

Comment:
Accordingly with https://getcomposer.org/doc/articles/versions.md#caret-version-range-,
in this way we are forcing php version to `>=7.1.0 <7.2.0` or `>=8.0 <8.1`. Is it the desired behavior?

Reply:
You are right, strange that it is not notified by the pipeline with PHP 7.4 and PHP 8.2 images

Reply:
We realized that pipeline does not fail since we are not installing the library itself but its dependencies.